mitsurunner.yaml

```diff
@@ -124,7 +124,7 @@ sensor:
             }
 
         // Jos haluat logata lämpötilaeroa, lisää defineen polku constants.h -tiedostoon
-        #ifdef TOPIC_TEMP_DELTA
+        #ifdef TOPIC_TEMP_DELTA_TRACES
```

Comment:
Itse olin käyttänyt tätä siis suoraan näin, jolloiin toimii ilman muutostsa, mutta voi olla selkeämpi, että kaikki topic-polut ovat const char* (tai std::string toiminee myös).
#define TOPIC_TEMP_DELTA "pub/qwer/asdf/zxcv/temp_delta"

Reply:
Joo, molemmin tavoin toimii. Menin harhaan, joten aina on selkeämpää jos asiat on mahdollista toteuttaa/tulkita vain yhdellä tavalla. Joten mergeän tämän muutoksen mainiin.